.env.template

```diff
@@ -50,6 +50,8 @@ SMART_TOKEN_LIMIT=8000
 
 # MEMORY_BACKEND - Memory backend type (Default: local)
 MEMORY_BACKEND=local
+# MEMORY_EMBEDER - Embeddings model to use (Default: ada)
+MEMORY_EMBEDER=ada
```

Comment:
EMBEDDER is the right spelling I guess, double D as in EMBEDDED

Reply:
Should be fixed with 2a147ac.